extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -136,7 +136,7 @@ export class AmpAdXOriginIframeHandler {
     // iframe.
     listenForOncePromise(this.iframe, 'entity-id', true)
         .then(info => {
-          this.element_.creativeId = info.data.id;
+          this.element_.creativeId = info.data['id'];
```

Comment:
No. Slightly confusingly `listenForOncePromise` returns an internal data structure that looks similar to a browser event, but isn't. It is also properly typed and so enforces the `[]` subscript automatically.